src/misc/lv_fs.c

```diff
@@ -97,6 +100,7 @@ lv_fs_res_t lv_fs_open(lv_fs_file_t * file_p, const char * path, lv_fs_mode_t mo
         const char * real_path = lv_fs_get_real_path(path);
         void * file_d = drv->open_cb(drv, real_path, mode);
```

Comment:
Maybe leave a LV_PROFILE_BEGIN_TAG("open_cb"); or something is ok

Reply:
I think `LV_PROFILER_BEGIN` is enough, and the time consuming of other operations is almost negligible. let's keep the code simple :)